eng/templates/variables/installer-pipelines.yml

```diff
@@ -1,12 +1,12 @@
-# Information about dotnet/installer pipelines that won't change and is useful in multiple pipelines
+# Information about dotnet/installer pipelines that wont change and is useful in multiple pipelines
```

Comment:
```suggestion
# Information about dotnet/installer and VMR pipelines that won't change and is useful in multiple pipelines
```

Reply:
Also, this file's name could be changed to `pipelines.yml` or `pipeline-info.yml` because it no longer contains only dotnet/installer's pipeline information.